swaymsg/swaymsg.1.scd

```diff
@@ -47,14 +47,14 @@ _swaymsg_ [options...] [message]
 	See **sway**(5) for a list of commands.
 
 	Tips:
-	- Command expansion is performed twice: once by swaymsg, and again by sway.
-	If you have quoted multi-word strings in your command, enclose the entire command
-	in single-quotes. For example, use _swaymsg 'output "Foobar Display" enable'_
-	instead of _swaymsg output "Foobar Display" enable_
-	- If you are proving a command that contains a leading hyphen (_-_),
-	insert two hyphens (_--_) before the command to signal to swaymsg not to
-	parse anything beyond that point as an option. For example, use
-	_swaymsg -- mark --add test_ instead of _swaymsg mark --add test_
+		- Command expansion is performed twice: once by swaymsg, and again by sway.
```

Comment:
The hyphen shouldn't be indented. You can test your changes with:

```sh
scdoc <swaymsg/swaymsg.1.scd | man -l -
```

Reply:
whoops, thanks. Now it renders better:

![image](https://user-images.githubusercontent.com/907257/87533505-008eb280-c695-11ea-82b7-b6f468fb8bcd.png)